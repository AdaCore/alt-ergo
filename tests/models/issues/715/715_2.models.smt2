(set-logic ALL)
(set-option :produce-models true)
(declare-const f Int)
(assert (= f 0))
; This function is user-defined and shouldn't appear in the output model.
(define-fun g ((x Int) (y Int)) Int (+ x y))
(check-sat)
(get-model)
