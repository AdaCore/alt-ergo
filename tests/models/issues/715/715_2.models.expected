
unknown
(
  (define-fun f () Int 0)
)
